src/index.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module 'type-to-reducer' {
+  export default function typeToReducer (
+    reducerMap: any,
```

Comment:
Maybe reducerMap should be an object of reducerMaps or functions rather than an any?